locales/locales.json

```diff
@@ -216,7 +216,6 @@
         "disclaimer": "ናይ ደራሲ ዝርዝራት ሓበሬታን ናይ ርክብን",
         "mainDisclaimer": "ናይ ደራሲ ዝርዝራት ሓበሬታን ናይ ርክብን - {{appName}}",
         "pois": "ዘገድሱ ነጥብታት",
-        "notFound": "ገጽ ኣይተረኽበን",
```

Comment:
I think all translators used german as their source. This means the "Page not located" is an exception.

Did you use google translator here? If yes then I woudl say we leave the translation here.

Reply:
Yes, google translates this as "Sagen Sie uns nicht die Seite" or "Don't tell us the page", that's why I deleted it. I'm not sure how to check this translation.

Reply:
Me neighter :D but I'm oke with removing if we are not sure :+1: